Week3/Homework/Js-exercises/amazingNoer.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
+
+let giveCompliment = function(name) {
+  let complimentsList = [
+    "respected",
+    "praiseworthy",
+    "awesome",
+    "handsome",
+    "heroic",
+    "noteworthy",
+    "intelligent",
+    "hardworking",
+    "humble",
+    "admirable"
+  ];
+  let getRandomNumber = [Math.floor(Math.random() * complimentsList.length)];
+  let randomCompliment = complimentsList[getRandomNumber];
+  console.log("You are " + randomCompliment + ", " + name + "!");
```

Comment:
1. `const` instead of `let` here please. Would suggest to make it a habit to always use `const` while creating variables and only if you need to reassign variable should you use `let`. 🙂 
2. I think you can return the statement instead of logging it on the console inside the function. Would be better.
3. Would like to introduce to you a new feature in JavaScript called template literals - https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Template_literals. You need not use + operator then to concatenate strings.